docs/reference/remap/functions/remove_datadog_tags.cue

```diff
@@ -0,0 +1,52 @@
+package metadata
+
+remap: functions: remove_datadog_tags: {
+	category: "String"
+	description: #"""
+		Remove a set of tags from an existing tags list that follows the Datadog tag format. Datadog logs tags should be kept in the `ddtags` fields to be properly accounted for if the
+		event is ultimately sent to Datadog using the `datadog_logs` sink. Tags are usually following the `key:value` convention thus the `remove_datadog_tags` can be used to remove all
```

Comment:
> Tags are usually following the `key:value` convention

So this isn't a _hard requirement_?

At first glance it seems `add_datadog_tags` doesn't require `:` to be the delimiter, but `remove_datadog_tags` does. That seems inconsistent to me. I think we should either:

1. require `:` as the delimiter in both functions, or
2. allow overriding the default `:` delimiter in `remove_datadog_tags`